buttons.html

```diff
@@ -14,9 +14,9 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
-    <link rel="stylesheet" href="framework.css">
-    <link rel="stylesheet" href="animations.css">
-    <link rel="stylesheet" href="reset.css">
+    <link rel="stylesheet" href="css/framework.css">
+    <link rel="stylesheet" href="css/animations.css">
+    <link rel="stylesheet" href="css/reset.css">
```

Comment:
@LarryMatte You should include the reset file as the first file, or you might end up with specificity problems on basic elements